src/main/java/com/direwolf20/buildinggadgets/common/tools/NBTTool.java

```diff
@@ -0,0 +1,321 @@
+package com.direwolf20.buildinggadgets.common.tools;
+
+import it.unimi.dsi.fastutil.ints.IntArrayList;
+import it.unimi.dsi.fastutil.ints.IntList;
+import net.minecraft.nbt.*;
+
+import javax.annotation.Nonnull;
+import java.util.Map;
+import java.util.function.Function;
+
+public class NBTTool {
```

Comment:
I know what's this is doing but could we have a docBlock on the util like classes?